xml/vt_scenarios.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" 
+xmlns:xi="http://www.w3.org/2001/XInclude" 
+xmlns:xlink="http://www.w3.org/1999/xlink" 
+version="5.0" xml:id="cha-virtualization-scenarios">
+<!-- I/O virtualization -->
+ <title>Virtualization Scenarios</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+   </dm:bugtracker>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Virtualization design provides several useful capabilities to your
+    organization: more efficient hardware use, support for legacy software,
+    operating system isolation, live migrations, disaster recovery, and 
+    load balancing. 
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec-virtualization-scenarios-capabilities">
+  <title>Server Consolidation</title>
+  <para>
+   Many servers can be replaced by one big physical
+   server, so hardware is consolidated, and guest operating systems are
+   converted to virtual machines. This also supports running legacy
+   software on new hardware.
+   <!--
+   reduce cost
+   dont waste space and resources not running at 100%
+   reduce servers locations
+   compute resources more efficiently (multiple workloads)
+   simplify the data center infrastructure
+   move workload easily to another host avoiding downtime
+   VM provisionning
+   multiple guest operating system on 1 host
+   no impact in migration
+
+   maintenance windows
+   disk capacity
+   server capabilities
+   -->
+  </para>
+ </sect1>
+ <sect1>
+  <title>Isolation</title>
+  <para>
+   A guest operating system can be fully isolated from the Host
+   running it. So if the virtual machine is corrupted, the Host system is
```

Comment:
is there a reason for Host with capital 'H' when guest remains with a small 'g'?
```suggestion
   A guest operating system can be fully isolated from the host
   running it. So if the virtual machine is corrupted, the host system is
```